__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+const db = require('../db/connection');
+const request = require('supertest');
+const app = require('../app');
+
+
+beforeEach(() => seed(testData));
+
+afterAll(() => db.end());
+
+describe("App", () => {
+    describe("GET /api/topics", () => {
+        it("Check if endpoint returns an array of topic objects", () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then((response) => {
+                expect(Array.isArray(response.body)).toEqual(true);
+                expect(typeof(response.body[0])).toEqual('object');
```

Comment:
Using toEqual is a brittle test for this endpoint because it doesn't leave room for extension. Please amend the testing here to be more flexible. This means using 'expect.objectContaining' or '.toMatchObject' instead 👍